src/open_inwoner/accounts/views/contacts.py

```diff
@@ -48,8 +49,17 @@ def get_context_data(self, **kwargs):
         context["contacts_for_approval"] = user.get_contacts_for_approval()
         context["pending_invitations"] = user.get_pending_invitations()
         context["form"] = ContactFilterForm(data=self.request.GET)
+        context["messages_enabled"] = self.messages_enabled
         return context
 
+    @property
+    def messages_enabled(self) -> bool:
+        """
+        :returns: True if a message page exists and is set to be published, False otherwise
+        """
+        message_page = Page.objects.filter(application_namespace="inbox").first()
```

Comment:
Let's use `application_namespace=InboxApphook.app_name` to be sure. 
I would add `publisher_is_draft=False` to the filter in order to get only the non-draft here.

https://docs.django-cms.org/en/latest/topics/publishing.html